source/ThemeManager.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+** Project FileShare, 2022
+**
+** Author Léo Lhuile
+**
+** Started on  Thu Jul 24 15:44:13 2025 Léo Lhuile
+** Last update Thu Jul 24 15:44:13 2025 Léo Lhuile
+**
+** ThemeManager.cpp : Implementation of ThemeManager class
+*/
+
+#include "ThemeManager.hpp"
+
+namespace FileShare::GUI {
+    ThemeManager::ThemeManager()
+    {
+        this->setTheme(this->currentTheme);
+    }
```

Comment:
This does not seem persistent, but I forgot - do we currently persist settings or not yet ?